src/pages/index.pug

```diff
@@ -23,6 +23,41 @@ block pageComponents
 
   section.section
     .container
+      +page-header({
+        pageHeaderHeading: "What is automation?",
+        pageHeaderText: "Automation is the act of using data, design and technology to turn a task, traditionally undertaken by humans, into a partially or completely automatic process."
+      })
+
+
+    .grid.grid--statements
+      .grid__item
+        +statement({
+          statementIcon: "/assets/images/icons/magic.svg",
+          statementHeading: "Business Processes",
+          statementText: "Automating repetitive and well-defined tasks, like running weekly reports or reviewing an application, keeps your team focused on doing the hard parts computers can't do well."
+        })
+
+      .grid__item
+        +statement({
+          statementIcon: "/assets/images/icons/tasks.svg",
+          statementHeading: "Higher data quality",
+          statementText: "Lookups and autocompleting fields make it easier for users to provide high-quality information, which itself creates further automation opportunities in processing and handling said data."
+        })
+
+      .grid__item
+        +statement({
+          statementIcon: "/assets/images/icons/growth.svg",
+          statementHeading: "Improved service",
+          statementText: "Create convenience by automating customer processes; automatic emails and online self-help are quicker and more cost-effective than hand-posted letters or physical 9-5 support teams."
```

Comment:
I think this is probably a good heading to drop a mention of digital products under:

> Digital products allow customers to self-serve  in convenience, with communications via their preferred medium, and entirely unconstrained by the 9-to-5.